control-plane/agents/src/bin/core/volume/service.rs

```diff
@@ -198,6 +227,19 @@ impl Service {
         Ok(())
     }
 
+    /// Destroy the shutdown targets associate with the volume.
+    #[tracing::instrument(level = "info", skip(self), err, fields(volume.uuid = %request.uuid))]
+    pub(super) async fn destroy_shutdown_target(
+        &self,
+        request: &DestroyShutdownTargets,
+    ) -> Result<(), SvcError> {
+        let mut volume = self.specs().volume(&request.uuid).await?;
+        volume
+            .remove_shutdown_targets(&self.registry, request)
+            .await?;
+        Ok(())
```

Comment:
keeping with the current style of the file:
```suggestion
        volume
            .remove_shutdown_targets(&self.registry, request)
            .await
```